.github/CONTRIBUTING.md

```diff
@@ -311,54 +316,58 @@ def test_explain_what_is_being_tested(tmpdir):
     # BoringModel is a functional model. You might want to set methods to None to test your behaviour
     # Example: model.training_step_end = None
 
-    trainer = Trainer(default_root_dir=tmpdir, ...)  # will save everything within a tmpdir generated for this test
+    trainer = Trainer(
+        default_root_dir=tmpdir, ...
+    )  # will save everything within a tmpdir generated for this test
```

Comment:
Does it set a line length? This is under 120 lines

Reply:
good catch, opening issue https://github.com/hukkin/mdformat-black/issues/2